src/components/mods/modsTable.tsx

```diff
@@ -111,8 +142,50 @@ export const ModsTable = ({ qualities, difficulties, modsWithInfo, isLoading }:
         [difficulties],
     );
 
+    const [currentTabIndex, setCurrentTabIndex] = useState(null as null | number);
```

Comment:
The better way to do this is to pass the optional type parameter to the useState function